test/kixi/integration/metadata_test.clj

```diff
@@ -3,16 +3,20 @@
             [clojure.test :refer :all]
             [clj-http.client :as client]
             [clojure.java.io :as io]
-            [kixi.integration.base :refer :all]
+            [kixi.integration.base :refer :all :exclude [post-file wait-for-metadata-key]]
             [kixi.datastore.metadatastore :as ms]
             [kixi.datastore.web-server :as ws]
             [kixi.datastore.schemastore.conformers :as conformers]
-            [kixi.datastore.schemastore :as ss]))
+            [kixi.datastore.schemastore :as ss]
+            [kixi.integration.base :as base]))
```

Comment:
Can you just add the `:as base` to the declaration on line 6 or is there a reason to keep it separate?